### DISCUSSION TITLE
Custom Code Snippets?

### DISCUSSION THREAD

**Discussion by hankbinkle** [phcode-dev/phoenix#1781] [Q&A]
Hello all,

I believe it was available in Atom, but can you set up custom key strokes to drop in commonly use code snippets in Phoenix? For example, I would start typing "cl" and it would dump this code into the editor.

`console.log(`foo: ${foo}`);`

Thanks!

**Comment by abose** [comment]
This is a pending  feature request we had: https://github.com/phcode-dev/phoenix/issues/1014

The workaround is to use extension `Brackets Snippets (by edc)` that does exactly what you wanted.  
![image](https://github.com/user-attachments/assets/24cb890a-48ba-4bd7-a086-8f60d97e391c)

But that appears to be not working in Linux/Mac desktop app. Let me check why the extension is not working there.

**Comment by hankbinkle** [comment]
ok thanks!

Thanks,

Rob

Rob Zinn
c:904.377.9854
***@***.***

Sent via Superhuman iOS ***@***.***>
